misc/bazel/registry/modules/rules_kotlin/1.9.4-codeql.1/patches/module_dot_bazel_version.patch

```diff
@@ -0,0 +1,12 @@
+===================================================================
+--- a/MODULE.bazel
++++ b/MODULE.bazel
+@@ -1,7 +1,7 @@
+ module(
+     name = "rules_kotlin",
+-    version = "1.9.0",
++    version = "1.9.4-patched",
```

Comment:
Should this be `1.9.4-codeql.1`?

Reply:
this is actually unneeded, let me remove it